packages/kit/src/compatibility.ts

```diff
@@ -2,6 +2,10 @@ import satisfies from 'semver/functions/satisfies.js' // npm/node-semver#381
 import type { Nuxt, NuxtCompatibility, NuxtCompatibilityIssues } from '@nuxt/schema'
 import { useNuxt } from './context'
 
+export function normaliseSemanticVersion (version: string) {
```

Comment:
It hurts me deeply but we should probably use American English here (and elsewhere where this function is used):

```suggestion
export function normalizeSemanticVersion (version: string) {
```